policyengine_us/parameters/gov/states/ga/tax/income/credits/itemizer/amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: Georgia provides this itemizer tax credit amount for resident taxpayers who itemize their deductions.
+values:
+  2021-01-01: 0
```

Comment:
No need to set it to 0 since we will only be adding it to the computation tree in 2024